merged_fs_test.go

```diff
@@ -103,6 +103,91 @@ func TestMergedFS(t *testing.T) {
 	t.Logf("Content of test1.txt: %s\n", string(content))
 }
 
+func TestReadFileFS(t *testing.T) {
+	zip1 := openZip("test_data/test_a.zip", t)
+	zip2 := openZip("test_data/test_b.zip", t)
+	zip3 := openZip("test_data/test_c.zip", t)
+
+	// Merge all three zip files into a single FS, with zip1 being highest
+	// priority, and zip3 being lowest priority.
+	var m fs.FS = NewMergedFS(zip1, NewMergedFS(zip2, zip3))
+	merged, ok := m.(fs.ReadFileFS)
+	if !ok {
+		t.Errorf("Expected MergedFS to implement fs.ReadFileFS interface.")
+		return
+	}
+	expectedFiles := []string{
```

Comment:
You can probably delete everything starting here (line 119) through line 159.  I understand that you want to call `fstest.TestFS` on the `ReadFileFS` instance, but fortunately the `fstest` package already includes special cases if the target FS implements the interface (for example, see [this line in the fstest code](https://cs.opensource.google/go/go/+/refs/tags/go1.22.0:src/testing/fstest/testfs.go;drc=6dc2d2aa6b9cbab9dc190276cb5a77b4b4a91cd3;l=531)).  Therefore, by implementing the ReadFileFS interface, the existing `TestMergedFS` function's call to `fstest.TestFS` will be sufficient for testing the new functionality.

(Though I still think it's a good idea to explicitly test that `test1.txt` produces the same contents when read through either method.)